trade-finance-manager-api/src/v1/controllers/auth/auth-service/map-entra-user-data.js

```diff
@@ -0,0 +1,28 @@
+const { getTfmRolesFromEntraGroups } = require('../../../helpers/get-tfm-roles-from-entra-groups');
+const { mapFirstAndLastName } = require('./map-first-and-last-name');
+const { DATE } = require('../../../../constants');
+
+/**
+ * Map Entra user data
+ * @param {Object} entraUser: Entra user data
+ * @param {Object} tfmUser: Optional TFM user object.
+ * @returns {Object} Mapped user data
+ */
+const mapEntraUserData = (entraUser, tfmUser) => {
```

Comment:
I don't know if we want to spend time typing this properly -- if so, we could look at TS and then get rid of the js docs